src/components/editMeasure/populationCriteria/groups/QICore/QICoreMeasureGroups.tsx

```diff
@@ -628,6 +628,13 @@ const MeasureGroups = (props: MeasureGroupProps) => {
             (measureGroupNumber === 0 ? 1 : measureGroupNumber)
         );
         handleDialogClose();
+        if (response && response.groups?.length > 0) {
+          setToastOpen(true);
+          setToastType("success");
+          setToastMessage(
+            "Measure criteria successfully deleted. Your Criteria's and populations have been re numbered."
```

Comment:
`'s` in Criteria's seems unnecessary.

Reply:
@adongare Ok thanks Ashok, I copied it from AC, but I think you are right, it's not necessary. I just removed it.